web/README.md

```
@@ -1,45 +1,43 @@
-   ```bash
-   npm run dev
-   ```
+```bash
+npm run dev
+```
 
-   _The following output confirms a successful launch of the system:_
+_The following output confirms a successful launch of the system:_
 
-   ![](https://github.com/user-attachments/assets/0daf462c-a24d-4496-a66f-92533534e187)
+![](https://github.com/user-attachments/assets/0daf462c-a24d-4496-a66f-92533534e187)
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)